backend/gn_module_monitoring/command/cmd.py

```diff
@@ -183,6 +184,29 @@ def cmd_install_monitoring_module(module_code):
     return
 
 
+@click.command("clean_existing_permissions")
+@click.argument("module_code", required=False, default="")
```

Comment:
Consider handling the case where `module_code` is not provided in the `cmd_clean_existing_permissions` function. Currently, if `module_code` is not provided, the function does nothing. This could lead to silent failures. [important]